stable/apm-server/templates/deployment.yaml

```diff
@@ -0,0 +1,91 @@
+{{- if eq .Values.kind "Deployment" }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "apm-server.fullname" . }}
+  labels:
+    app: {{ template "apm-server.name" . }}
+    chart: {{ template "apm-server.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  minReadySeconds: 10
+  strategy:
+{{ toYaml .Values.updateStrategy | indent 4 }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "apm-server.name" . }}
+        release: {{ .Release.Name }}
```

Comment:
Could you please use the metadata outlined [here](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata). This is based on Kubernetes documented ones to promote interoperability. This applies everywhere.